### PR TITLE
Update the documentation of the HTTP module

### PR DESCRIPTION
**Pull request by LaszloLango**
IoT.js-DCO-1.0-Signed-off-by: László Langó llango.u-szeged@partner.samsung.com

### CODE REVIEW

**Comment by glistening** [comment]
@LaszloLango Could you check my comment in #873?

**Comment by LaszloLango** [comment]
@glistening @daeyeon, there is no platform specific part in HTTP. Should I add "Platform Support" section or not?

**Comment by daeyeon** [comment]
@LaszloLango Yes, it's needed.

**Comment by LaszloLango** [comment]
@glistening @daeyeon I've updated the PR according to #873. Please check.

File: docs/api/IoT.js-API-HTTP.md [hunk 1]
```diff
@@ -1,356 +1,327 @@
...
-<a id="markdown-statusmessage" name="statusmessage"></a>
-### `statusMessage`
+### message.statusMessage
 HTTP response status message as `String`
 
-<a id="markdown-socket" name="socket"></a>
-### `socket`
-Underlying socket
+### message.url
+Requests URL as `String`
```
Comment (daeyeon) [inline docs/api/IoT.js-API-HTTP.md]: String -> string. sorry for asking fixing minor mistakes.

File: docs/api/IoT.js-API-HTTP.md [hunk 2]
```diff
@@ -1,356 +1,327 @@
...
-<a id="markdown-url" name="url"></a>
-### `url`
-Requests URL as `String`
+### message.socket
+Underlying socket
 
-<a id="markdown-statuscode" name="statuscode"></a>
-### `statusCode`
+### message.statusCode
 HTTP response status code as `Number` of 3-digit.
```
Comment (daeyeon) [inline docs/api/IoT.js-API-HTTP.md]: Number -> number.

**Comment by yichoi** [comment]
LGTM

**Pull request merged**